app/views/users/_postcode_field.html.haml

```diff
@@ -1,11 +1,12 @@
 %label.mb-0 Or find my constituency using my postcode
 
-.form-group.form-group-short
-  .input-group
-    %input.form-control#txt-postcode{ type: "text", size: 10, minlength: 5,
-                                      maxlength: 9, spellcheck: "false" }
-    .input-group-append
-      %input.btn.btn-secondary#btn-postcode{ type: "button", value: "Search" }
+.d-flex.flex-row.justify-content-center
```

Comment:
I don't think this is right. We don't want to always centre the postcode field. Instead I think you need to do what I suggested here?

https://github.com/swapmyvote/swapmyvote/pull/1024/files#r1835562684